.ruby-style.yml

```diff
@@ -807,34 +807,34 @@ Style/RegexpLiteral:
 Style/RescueModifier:
   Description: 'Avoid using rescue in its modifier form.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-rescue-modifiers'
-  Enabled: false
+  Enabled: true
 
 Style/SelfAssignment:
   Description: >-
                  Checks for places where self-assignment shorthand should have
                  been used.
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#self-assignment'
-  Enabled: false
+  Enabled: true
 
 Style/Semicolon:
   Description: "Don't use semicolons to terminate expressions."
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#no-semicolon'
-  Enabled: false
+  Enabled: true
 
 Style/SignalException:
   Description: 'Checks for proper usage of fail and raise.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#prefer-raise-over-fail'
-  Enabled: false
+  Enabled: true
 
 Style/SingleLineBlockParams:
   Description: 'Enforces the names of some block params.'
   StyleGuide: 'https://github.com/bbatsov/ruby-style-guide#reduce-blocks'
-  Enabled: false
+  Enabled: true
```

Comment:
We tend to use `memo` instead of `acc`. I beelive it was you who got me into that habit, do we want to change?

See: https://rubocop.readthedocs.io/en/latest/cops_style/#stylesinglelineblockparams

Reply:
Ah, hmm yeah, I've always used `memo` because that's what Ruby Core docs tend to use. Maybe we should drop this one; I had initially thought it was more about e.g. using [underscores for unused params](https://github.com/bbatsov/ruby-style-guide#reduce-blocks).

Reply:
Removed and added https://github.com/rubocop-hq/ruby-style-guide#no-cryptic-perlisms instead